binding/src/test/java/se/fortnox/reactivewizard/binding/AutoBindModulesTest.java

```diff
@@ -28,12 +28,12 @@ void shouldBindImplementationsToTheirInterfaces() {
     }
 
     @Test
-    void shouldUseBootStrapBindingOverDefault() {
```

Comment:
This seems to be a very conscious choice, did you find any hints on why it was this way? It could help prevent reintroducing bugs that this might have resolved.

Reply:
No, I don't have any clue why it's done this way. The bootstrap module binds only two things:
``` java
                    bind(String[].class).annotatedWith(Names.named("args")).toInstance(args);
                    bind(ConfigFactory.class).toInstance(configFactory);
``` 
So it's either the default ConfigFactory class or whatever ConfigFactory compatible class defined in the override modules. I could not find hints why the default factory should always override.

Reply:
Maybe this order of overrides was made just for a single purpose: to promote the TestInjector in DbProxyTest. When this case is fixed, both tests and applications work fine.